lib/kracken/controllers/token_authenticatable.rb

```diff
@@ -14,7 +14,7 @@ def self.included(base)
       end
 
       # NOTE: Monkey-patch until this is merged into the gem
-      def request_http_token_authentication(realm = 'Application')
```

Comment:
How will this work for those apps running on Rails 4? I guess because it's the 2nd param and it's optional it will work but just be `nil`?